src/common/ant.ts

```diff
@@ -119,6 +190,21 @@ export class AoANTReadable implements AoANTRead {
   async getInfo(
     { strict }: AntReadOptions = { strict: this.strict },
   ): Promise<AoANTInfo> {
+    if (await this.checkHyperBeamPromise) {
+      const state = await this.getState();
+      return {
+        Name: state.Name,
+        Ticker: state.Ticker,
+        Description: state.Description,
+        Keywords: state.Keywords,
+        Denomination: state.Denomination.toString(),
+        Owner: state.Owner,
+        Logo: state.Logo,
+        'Total-Supply': state.TotalSupply.toString(),
+        Handlers: [], // TODO: support for handler in patched state
```

Comment:
@atticusofsparta - do we use Handlers anymore?

Reply:
We do but shouldnt. Its been on my list to make a feature flag map and list it along with every version mapped to the module ID in the ant registry. Needing the handlers means that we need to do another call to check api support for enabling features in the portal. I would rather load the handlers based on the version and then be able to have the ANT class know whether or not to throw based on its handlers state.